File: simData.jl
using Distributions
using DataFrames
using CSV

α = .4
β = .1

lambdas = rand(Gamma(α, β), 8)

data = zeros(500, 8)

for i in 1:8
  data[:,i] = rand(Exponential(lambdas[i]), 500)
end

CSV.write("simExpData2.csv", convert(DataFrame, data))

File: simData.R

# Create Simulated data

alpha <- .4
beta <- .1

lambdas <- rgamma(8, alpha, beta)

simData <- c()

for(i in 1:length(lambdas)){
  simData <- cbind(simData, rexp(500, lambdas[i]))
}

write.csv(simData, "simExpData.csv")
